src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -398,6 +401,72 @@ def remove_temp_test_dir(self):
             errors.append("Error removing temporary test files: {}".format(error))
         return errors
 
+    def _cleanup(self):
+        """Run all the the cleanup methods from last to first.
+
+        Returns:
+            list: a list of error strings to report at the end of tearDown().
+
+        """
+        errors = []
+        while self._cleanup_methods:
+            try:
+                cleanup = self._cleanup_methods.pop()
+                errors.extend(cleanup["method"](**cleanup["kwargs"]))
+            except Exception as error:      # pylint: disable=broad-except
+                kwargs_str = ", ".join(
+                    ["=".join([str(key), str(value)]) for key, value in cleanup["kwargs"].items()])
+                errors.append(
+                    "Unhandled exception when calling {}({}): {}".format(
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string